### PR TITLE
[Doc] add starrocks kubernetes operator release note

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Why I'm doing:
Several versions of kubernetes operator and helm are released, but the relevant release note is missing on our doc site.
What I'm doing: 
Add the release note for kubernetes operator and helm.

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Review by amber-create — DISMISSED** [review]

File: docs/ecosystem_release/starrocks-kubernetes-operator.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  - Default configuration file for StarRocks Operator: `operator.yaml`
+  - Helm Chart, including `kube-starrocks` Chart `kube-starrocks-${chart_version}.tgz`. The `kube-starrocks` Chart is divided into two subcharts: `starrocks` Chart `starrocks-${chart_version}.tgz` and `operator` Chart `operator-${chart_version}.tgz`.
+
+For example, the download URL for kube-starrocks chart v1.8.6 is: [kube-starrocks](https://github.com/StarRocks/starrocks-kubernetes-operator/releases/download/v1.8.6/kube-starrocks-1.8.6.tgz)
+
+**Version requirements**
+
+Kubernetes: 1.18 or later
+
+Go: 1.19 or later
```
Comment (amber-create) [inline docs/ecosystem_release/starrocks-kubernetes-operator.md:35]: image
It's better to use an unordered list.

File: docs/ecosystem_release/starrocks-kubernetes-operator.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+## 1.8
+
+**1.8.6**
+
+**Bug Fixes**
+
+Fixed the following issue:
+
+- An error  `sendfile() failed (32: Broken pipe) while sending request to upstream` is return during a Stream Load job. After Nginx sends the request body to FE, FE then redirects the request to BE. At this point, the data cached in Nginx may already be lost. [#303](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/303)
```
Comment (amber-create) [inline docs/ecosystem_release/starrocks-kubernetes-operator.md:47]: image
-> is returned

Comment (amber-create) [inline docs/ecosystem_release/starrocks-kubernetes-operator.md:47]: image
-> the FE

Comment (amber-create) [inline docs/ecosystem_release/starrocks-kubernetes-operator.md:47]: image
-> the BE

File: docs/ecosystem_release/starrocks-kubernetes-operator.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+**Doc**
+
+- [Load data from outside the K8s network to StarRocks through FE proxy](https://github.com/StarRocks/starrocks-kubernetes-operator/blob/main/doc/load_data_using_stream_load_howto.md)
```
Comment (amber-create) [inline docs/ecosystem_release/starrocks-kubernetes-operator.md:51]: image
Use the full spelling. Check the entire doc.

File: docs/ecosystem_release/starrocks-kubernetes-operator.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+**Bug Fixes**
+
+- **[Helm Chart]** The root user's password in StarRocks may not be initialized successfully when `starrocks.initPassword.enabled` is true and the value of `starrocks.starrocksCluster.name` is specified. It is caused by the wrong FE service domain name used by the initpwd pod to connect FE service. More specifically, in this scenario, FE service domain name uses the value specified in `starrocks.starrocksCluster.name`, while the initpwd pod still uses the value of the `starrocks.nameOverride` field to form the FE service domain name. ([#292](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/292))
+
+**Upgrade notes**
+
+- **[Helm Chart]** When the value specified in `starrocks.starrocksCluster.name` is different from the value of `starrocks.nameOverride`, the old configmaps for FE, BE, and CN will be deleted. New configmaps with the new name for FE, BE, and CN will be created. **This may result in the restart of FE/BE/CN pods.**
```
Comment (amber-create) [inline docs/ecosystem_release/starrocks-kubernetes-operator.md:67]: image
-> with new names for the FE, BE, and CN
-> the FE, BE, and CN pods

File: docs/ecosystem_release/starrocks-kubernetes-operator.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+- Add more user guides on how to manage the StarRocks cluster. For example, how to configure [logging and related fields](https://github.com/StarRocks/starrocks-kubernetes-operator/blob/main/doc/logging_and_related_configurations_howto.md) and [mount external configmaps or secrets](https://github.com/StarRocks/starrocks-kubernetes-operator/blob/main/doc/mount_external_configmaps_or_secrets_howto.md). For more user guides, see [docs](https://github.com/StarRocks/starrocks-kubernetes-operator/tree/main/doc).
+
+**1.8.3**
+
+**Upgrade notes**
+
+- **[Helm Chart]** Add `JAVA_OPTS_FOR_JDK_11` to the default **fe.conf** file. When the default **fe.conf** file is used and the helm chart is upgraded to v1.8.3, **FE Pods may restart**. [#257](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/257)
```
Comment (amber-create) [inline docs/ecosystem_release/starrocks-kubernetes-operator.md:94]: image
"pods" or "Pods"? Stick to one usage across the doc. "pods" makes more sense to me.

**Review by amber-create — DISMISSED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Pull request merged**